custom_components/kia_uvo/KiaUvoApiEU.py

```diff
@@ -443,6 +443,27 @@ def get_geocoded_location(self, lat, lon):
         response = response.json()
         return response
 
+    def update_location(self, token: Token):
+        url = self.SPA_API_URL + "vehicles/" + token.vehicle_id + "/location"
+        headers = {
+            "Authorization": token.access_token,
+            "Stamp": token.stamp,
```

Comment:
```suggestion
            "Stamp": self.get_stamp(),
            "ccsp-service-id": self.CCSP_SERVICE_ID,
            "ccsp-application-id": self.APP_ID,
```

Reply:
I'm not certain that all these headers and the change are needed, but this aligns the request with the one from `update_vehicle_status` below.

Reply:
I don't see get_stamp() as a function?

Reply:
`get_stamp()` was introduced in 6fec63beadcdb07bebea6351773f1ea3c5b7e29d on Feb 8, so you'll need to bring this branch up to date for my suggestion to be applicable.

Reply:
Good catch. Thank you!